src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -639,15 +639,22 @@ public CloseDatabaseAction(LibraryTabContainer tabContainer, StateManager stateM
             this(tabContainer, null, stateManager);
         }
 
+        public TabPane getTabbedPane() {
+            return tabbedPane;
+        }
```

Comment:
Inversion of control. Do not add getters for components inside other components. They have to be injected.